registrations/utils.py

```diff
@@ -10,7 +13,11 @@ def normalise_msisdn(msisdn):
 
 
 def get_rapidpro_contact(msisdn):
-    contact = tembaclient.get_contacts(urn="tel:%s" % msisdn).first()
+    try:
+        contact = tembaclient.get_contacts(urn="tel:%s" % msisdn).first()
+    except TembaException as e:
+        logging.exception("Error connecting to RapidPro (msisdn: %s)" % msisdn)
+        raise e
```

Comment:
I'm wondering if we should do the same only alert on consecutive failures here. Or maybe we deploy and see if it's an issue before we do that.

Reply:
Yeah, I'd rather alert every time unless that turns out to be problematic